amundsen_application/static/js/components/common/ResourceListItem/styles.scss

```diff
@@ -20,32 +20,5 @@
   .content {
     width: 100%;
     min-width: 0; /* Needed to support `white-space: nowrap` */
-
-    .main-title {
-      color: $text-dark;
-      font-size: $font-size-large;
-      font-family: $font-family-sans-serif-bold;
-      font-weight: $font-weight-sans-serif-bold;
-    }
-
-    .secondary-title {
-      color: $text-medium;
-      font-family: $font-family-sans-serif-bold;
-      font-weight: $font-weight-sans-serif-bold;
-    }
-
-    .description {
-      color: $text-medium;
```

Comment:
Looks like there was some styling lost here that I don't see covered elsewhere.

Reply:
Changed 'body-3' to `body-3-secondary' with the medium font color.